Tallies/TallyClerks/tallyClerk_inter.f90

```diff
@@ -57,6 +57,7 @@ module tallyClerk_inter
   !!   reportOutColl    -> Process an outgoing from collision report
   !!   reportPath       -> Process pathlength report
   !!   reportTrans      -> Process transition report
+  !!   reportSpawn      -> Process fission report
```

Comment:
We don't really generate a new particle for N2N though, do we? I thought we just modify the weight, in which case I wouldn't call reportSpawn.

Reply:
No, no we do not at all :-) 

Sorry I just gave N2N as an example in case it was treated analogue at some point. I am in total agreement that there should be no spawn report when only the weight is changed. It should be only called when new, independent particle is created.